src/utils/hooks/api/vaults/get-vault-data.ts

```diff
@@ -37,22 +44,46 @@ interface VaultData {
     usd: number;
   };
   vaultAtRisk: boolean;
+  vaultStatus: string;
+  liquidationThreshold: Big;
+  premiumRedeemThreshold: Big;
+  secureThreshold: Big;
 }
 
-const getVaultOverview = async (
-  vault: VaultExt,
-  accountId: AccountId,
-  prices: Prices | undefined
-): Promise<VaultData> => {
+// TODO: move these to dictionary file
```

Comment:
This will be handled along with the rest of the dashboard text content (headings, CTAs etc.)